### PR TITLE
Implement MintableBurnable preset

### LINKED ISSUE #9: Implementation of Burnable and Mintable extensions
**Description**
---
Implement interface and functions to enable the minting and burning of tokens.

The list of functions are :
- `mint`
- `mintValue`
- `burn`
- `burnValue`

**Acceptance Criteria**
---
- [x] Implement the contents of each functions and events 
- [x] Implement unit tests for each functions
- [x] Implement integration tests for each functions

### PR DESCRIPTION
**Pull request by tekkac**
close #9

### CODE REVIEW

File: src/carbonable/erc3525/library.cairo [hunk 1]
```diff
@@ -295,6 +299,10 @@ namespace ERC3525 {
     func __mint_token{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
```
Comment (bal7hazar) [inline src/carbonable/erc3525/library.cairo:299]: double `_` prefix?

**Review by bal7hazar — DISMISSED** [review]
LGTM, 1 additional comment:

[library](https://github.com/Carbonable/carbonable-contracts/pull/14/files#diff-e208bfbcee225501ddaf04b3c4a4efb4fa6fe55c84c464b2ee94e38e0d0ba5f1R379):

Should we consider to burn the value before it burns a token, especially if the token can be reminted?

File: src/carbonable/erc3525/library.cairo (hunk 1 above)
Comment (tekkac) [reply src/carbonable/erc3525/library.cairo:299]: Yes, I took the same terminology as in the Solidity implementation. I guess a doubly '__' private method can *only* be called inside a single '_'  private method.

**Review by cloudvenger — DISMISSED** [review]

**Review by bal7hazar — APPROVED** [review]
LGTM

**Review by cloudvenger — APPROVED** [review]

**Pull request merged**